### PR TITLE
 Rename coin to token

### PR DESCRIPTION
**Pull request by brandonwestcott**
The `v0.1.1` release of tupelo renames coin to token. This updates the js client to match.

### CODE REVIEW

File: test/tokens.js [hunk 1]
```diff
@@ -39,24 +39,24 @@ describe("token operations", function() {
...
-      assert.fail("Should not be able to mint more tokens than establishCoin maximum")
+      await wallet.mintToken(chainId, walletKey, "token-a", 100);
+      assert.fail("Should not be able to mint more tokens than establishToken maximum")
     } catch (err) {
       assert.notEqual(err, null);
     }
 
     // FIXME: Should pass, fix incoming in tupelo
-    // resp = await wallet.mintCoin(chainId, walletKey, "token-a", 50);
+    // resp = await wallet.mintToken(chainId, walletKey, "token-a", 50);
```
Comment (cap10morgan) [inline test/tokens.js]: This could be uncommented now, couldn't it? (Perhaps not in this PR, though.)

Comment (brandonwestcott) [reply test/tokens.js]: yes, good call

Comment (brandonwestcott) [reply test/tokens.js]: Actually looks like thats dependent on your draft PR for tokens - seems the PR i had done to fix this didn't get carried over into the tupelo-go-client.

Comment (cap10morgan) [reply test/tokens.js]: Oh, right, b/c the mint stuff wasn't defined all the way out to the RPC interface yet. 👍

**Pull request merged**